installer/pkg/cmd/service_catalog.go

```diff
@@ -160,6 +160,10 @@ func installServiceCatalog(ic *InstallConfig) error {
 
 	err = deployConfig(dir)
 	if err != nil {
+		if strings.Contains(err.Error(), "\"etcd-operator\" is forbidden: attempt to grant extra privileges") {
+			fmt.Println("WARNING: Please run `kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value account)` before `sc install`.")
```

Comment:
"Warning: please run ..." ? Please double-check the style of `kubectl` that we seem to follow for `sc`.